File: app/src/main/java/com/neome/java/api/core/base/dto/DtoChatMessageOffset.java
// neome.ai API - do not change
//

package com.neome.java.api.core.base.dto;

import com.neome.java.api.meta.base.Types.ChatId;
import com.neome.java.api.meta.base.Types.MetaIdSpreadsheet;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DtoChatMessageOffset
{
  @Nullable
  public ChatId chatId;

  @Nullable
  public Long offset;

  @Nullable
  public MetaIdSpreadsheet spreadsheetId;
}

File: app/src/main/java/com/neome/api/core/base/dto/DtoChatMessageOffset.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.base.dto

import com.neome.api.meta.base.Types.ChatId
import com.neome.api.meta.base.Types.MetaIdSpreadsheet

interface DtoChatMessageOffset
{
  val chatId: ChatId?
  val offset: Long?
  val spreadsheetId: MetaIdSpreadsheet?
}